app/src/main/AndroidManifest.xml

```diff
@@ -17,12 +17,12 @@
         android:supportsRtl="true"
         android:usesCleartextTraffic="true"
         android:networkSecurityConfig="@xml/network_security_config"
-        android:theme="@style/AppTheme">
+        android:theme="@style/AppTheme.SplashScreen">
```

Comment:
If the user has selected the dark theme then the splash screen still has a light background.
[Peek 2023-01-31 18-21.webm](https://user-images.githubusercontent.com/14895212/215835615-4a4bcbcb-245d-4563-856d-55ea2cbb1027.webm)

Reply:
That's what I thought, didn't have the time to test it yet.

What about the Android version? Which API level are you running this on?

Reply:
I've tested it with 25 and 33 not sure which on which I tested this.

Reply:
I see, have you tested it on a physical device as well?
I suppose you would not have that problem there. But no guarantee.

Reply:
I will take a look. You can change the background color of the SplashScreen, but it is pre-defined in xml and not dynamically changeable I think.

Reply:
After some investigation I found out that this is the expected behaviour.
The native splashscreen background seems to adapt to your system light/dark mode.
Everything afterwards of course follows the in-app setting.

Google Maps and Google Chrome work like this as well.
See here (system dark mode, in-app light mode):

https://user-images.githubusercontent.com/46033730/216186950-ca1c63fb-4d25-49b1-b35d-37538edf9967.mp4

Reply:
Thanks for investigating, the behavior is okay then.